go/vt/vttablet/onlineddl/executor.go

```diff
@@ -967,13 +999,13 @@ func (e *Executor) cutOverVReplMigration(ctx context.Context, s *VReplStream, sh
 	if err != nil {
 		return vterrors.Wrapf(err, "failed getting locking connection")
 	}
+	defer lockConn.Recycle()
```

Comment:
This is an unrelated change that I spotted, and will also make a separate PR just for this change.

Reply:
Reference: https://github.com/vitessio/vitess/pull/17207